src/c2goto/library/stdlib.c

```diff
@@ -226,18 +226,23 @@ __ESBMC_HIDE:;
 }
 
 /* one plus the numeric value, rest is zero */
-static const unsigned char ATOI_MAP[256] = {
-  ['0'] = 1,
-  ['1'] = 2,
-  ['2'] = 3,
-  ['3'] = 4,
-  ['4'] = 5,
-  ['5'] = 6,
-  ['6'] = 7,
-  ['7'] = 8,
-  ['8'] = 9,
-  ['9'] = 10,
-};
+static const unsigned char get_atoi_map(unsigned char pos)
+{
+__ESBMC_HIDE:;
+  const unsigned char ATOI_MAP[256] = {
```

Comment:
Shouldn't this have static storage duration just like before?

Reply:
You're correct, @intrigus-lgtm. I have just pushed this commit https://github.com/esbmc/esbmc/commit/f37520d3bdd8f320a1d4188aced24d6c23d9c8c3.